src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -416,52 +416,6 @@
 					</VisualState.Setters>
 				</VisualState>
 			</VisualStateGroup>
-			<VisualStateGroup>
```

Comment:
?

Reply:
I couldn't build without removing this state group.

Reply:
It shouldn't have anything to do with this PR, are you getting the exception in `main` as well?

Reply:
Let me revert and see what will happened.

Reply:
Did it work?